demisto_sdk/commands/validate/tests/test_tools.py

```diff
@@ -416,7 +416,9 @@ def create_incident_type_object(
 
 
 def create_incident_field_object(
-    paths: Optional[List[str]] = None, values: Optional[List[Any]] = None
+    paths: Optional[List[str]] = None,
+    values: Optional[List[Any]] = None,
+    pack_info: Optional[Dict[str, Any]] = None,
```

Comment:
is this how we do it elsewhere? Perhaps passing a `Pack` object instead will be cleaner? Wdyt @YuvHayun?

Reply:
we do it in several other places, Idm doing as you proposed but let's change this for all relevant places.